src/sparse_blas/backends/cusparse/cusparse_handles.cpp

```diff
@@ -22,6 +22,7 @@
 #include "cusparse_error.hpp"
 #include "cusparse_helper.hpp"
 #include "cusparse_handles.hpp"
+#include "cusparse_scope_handle.hpp"
```

Comment:
Why is a cuSPARSE change coming in here?

Reply:
Is it related to the `common_launch_task.hpp` to keep common code between cuSPARSE and rocSPARSE backends?

Reply:
It's just as you said, `cusparse_task.hpp` used to include `cusparse_scope_handle.hpp` but it's not needed anymore (with the already merged changes to use the ScopeHandle object less often) and not possible anymore with the introduction of the more generic `common_launch_task.hpp`.